lib/ansible/modules/packaging/os/portage.py

```diff
@@ -326,13 +331,20 @@ def emerge_packages(module, packages):
         module.fail_json(msg='Use only one of usepkg, usepkgonly')
 
     emerge_flags = {
-        'jobs': '--jobs=',
-        'loadavg': '--load-average ',
+        'jobs': '--jobs',
+        'loadavg': '--load-average',
```

Comment:
turns out, it still doesn't work for `0`, because `0` is turned into `float(0.0)` and then turned into `str(0.0)`.